src/main/java/com/icthh/xm/ms/entity/service/impl/XmEntityServiceImpl.java

```diff
@@ -113,7 +114,7 @@
  * Service Implementation for managing XmEntity.
  */
 @Slf4j
-@LepService(group = "service.entity", name = "default")
+@LepService(group = "service.entity")
```

Comment:
@osiievska23 i sugggest do not touch this  (i mean name = "default"))

Reply:
ok, but
![image](https://github.com/user-attachments/assets/24661312-98c2-4387-8f5c-d53caddba9ca)

Reply:
done